enzyme/Enzyme/GradientUtils.h

```diff
@@ -79,6 +79,13 @@ extern llvm::StringMap<std::function<llvm::Value *(
     shadowHandlers;
 
 class DiffeGradientUtils;
+
+/// This is the main entry point to register a custom derivative for language frontends.
```

Comment:
I wouldn't say the main way, but perhaps "how to register custom derivatives programmatically"?